Project.toml

```diff
@@ -35,13 +35,14 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 AbstractMCMC = "0.5.2"
 AdvancedHMC = "0.2.20"
 AdvancedMH = "0.4"
-Bijectors = "0.4.0, 0.5, 0.6"
+Bijectors = "0.6.2"
 BinaryProvider = "0.5.6"
-Distributions = "0.21.11, 0.22"
-DistributionsAD = "0.4.1"
+Distributions = "0.22"
+DistributionsAD = "0.4.3"
```

Comment:
Consider
```suggestion
DistributionsAD = "~0.4.3"
```

Reply:
Done

Reply:
I realized too late that you discussed this change, but actually both versions are equivalent: https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1